src/api/undeploy/helpers/remove-deployment-file.js

```diff
@@ -0,0 +1,39 @@
+import { config } from '~/src/config/index.js'
+import { deleteGithubFile } from '~/src/api/undeploy/helpers/github/delete-github-file.js'
+
+const deploymentRepo = config.get('github.repos.appDeployments')
+const gitHubOwner = config.get('github.org')
+
+/**
+ * @typedef {import("pino").Logger} Logger
+ * @param {string} undeploymentId
+ * @param {string} imageName
+ * @param {string} environment
+ * @param {string} zone
+ * @param {{id: string, displayName: string}} user
+ * @param {Logger} logger
+ */
+async function removeDeploymentFile(
+  undeploymentId,
+  imageName,
+  environment,
+  zone,
+  user,
+  logger
```

Comment:
how come logger is being passed along? Can you not just used `createLogger()`?

Reply:
Yes though I thought we'd want to keep request id etc?